<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 242, 255)</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCustomPlot {background-color: #ffffff;
border-radius: 5%;
margin: 2;
}</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xMin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="yMin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xMax">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="yMaxLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>355</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
padding-left: 2;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Введите диапазон Y:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="yMax">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="xMaxLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>355</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: Black;
padding-left: 2;
font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
   </property>
   <property name="text">
    <string>Введите диапазон Х:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGraph">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Построить</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>result</zorder>
  <zorder>widget</zorder>
  <zorder>xMin</zorder>
  <zorder>yMin</zorder>
  <zorder>xMax</zorder>
  <zorder>yMaxLabel</zorder>
  <zorder>yMax</zorder>
  <zorder>xMaxLabel</zorder>
  <zorder>buttonGraph</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>graph/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
